src/aiokatcp/sensor.py

```diff
@@ -54,13 +54,31 @@
     overload,
 )
 
+try:
+    from typing import Protocol
+except ImportError:
+    # In Python 3.7 it's elsewhere. Mypy after 3.7 doesn't like this.
+    from typing_extensions import Protocol  # type: ignore
```

Comment:
No need to make this conditional. Just import from `typing_extensions` unconditionally (see client.py for example).